src/app/Jobs/ConvertVideoForStreaming.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Models\Video;
+use Carbon\Carbon;
+use Exception;
+use FFMpeg\Coordinate\Dimension;
+use FFMpeg\Format\Video\X264;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use ProtoneMedia\LaravelFFMpeg\Exporters\EncodingException;
+use ProtoneMedia\LaravelFFMpeg\Support\FFMpeg;
+
+class ConvertVideoForStreaming implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    public $video;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Video $video)
+    {
+        $this->video = $video;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        try {
+            FFMpeg::fromDisk($this->video->disk)
+            ->open($this->video->path)
+
+            // We are updating the size of the video to 720p
+            ->addFilter(function ($filters) {
+                $filters->resize(new Dimension(720, 480));
```

Comment:
This detail should be documented, the resizing covers 16:9 aspect ratios. If other ones are used the video will look distorted.

Non-breaking, however as a stretch goal we could have it resize to a specific width or height, then automatically compute the other one based on the aspect ratio.